main.go

```diff
@@ -213,7 +216,15 @@ func main() {
 }
 
 func runMany(dbpath string, getPort func() int, many, bucketSize, bsCon int, relay bool, stagger time.Duration) {
-	ds, err := levelds.NewDatastore(dbpath, nil)
+	dstore, err := levelds.NewDatastore(dbpath, nil)
+	if err != nil {
+		panic(err)
+	}
+
+	opts := pstoreds.DefaultOpts()
+	opts.CacheSize = 0
```

Comment:
Wont this increase our allocation velocity?

Reply:
Yes but in a shared setup, the cache is useless because it'll just be thrashing all the time.

Reply:
Depending on size and type of cache.

Reply:
We should have metrics regarding the hit rate. Unfortunately, open census was disabled because it was allocating too much (Why knows more).